connbeat.yml

```diff
@@ -9,6 +9,9 @@ connbeat:
   enable_docker: false
   docker_environment: ["PATH", "MESOS_TASK_ID"]
 
+processors:
+  - add_cloud_metadata:
```

Comment:
Should it be a key without any value? If so, looks good to me 👍

Reply:
This came directly from: https://github.com/raboof/connbeat/blob/master/connbeat.full.yml